Streetcode/Streetcode.BLL/MediatR/AdditionalContent/Subtitle/GetByStreetcodeId/GetSubtitlesByStreetcodeIdQueryHandler.cs

```diff
@@ -19,14 +19,14 @@ public GetSubtitlesByStreetcodeIdQueryHandler(IRepositoryWrapper repositoryWrapp
 
         public async Task<Result<IEnumerable<SubtitleDTO>>> Handle(GetSubtitlesByStreetcodeIdQuery request, CancellationToken cancellationToken)
         {
-            var subtitle = await _repositoryWrapper.SubtitleRepository.GetAllAsync(Subtitle => Subtitle.StreetcodeId == request.streetcodeId);
-            if (subtitle is null)
+            var subtitles = await _repositoryWrapper.SubtitleRepository.GetAllAsync(Subtitle => Subtitle.StreetcodeId == request.streetcodeId);
+            if (subtitles is null)
```

Comment:
Fix spacing
- Insert blank line above null guard
- Put repositiory meth call on the separate line
**And please, in future, try to be more responsible in putting punctuation together.
Do not forget that you're working in a team, and often other people besides yourself are reviewing your code and working upon its basis. I'm referring to both BE & FE.**

Reply:
1. Done
2. Updated. Basically, this template was written not by me, so I do not think that it is my fault.  This requires to be changed everywhere across BE then